runbot_travis2docker/models/runbot_build.py

```diff
@@ -178,6 +178,11 @@ def get_docker_run_cmd(self):
                 '-e', ('WEBLATE_HOST=%s' %
                        build.branch_id.repo_id.weblate_url)
             ]
+            if build.branch_id.repo_id.weblate_languages:
+                wl_cmd_env += [
+                    '-e', 'LANG_ALLOWED=%s' %
```

Comment:
Question:
MQT support use this parameter for just sync this items but what about if there is other .po... we need change MQT to process this list plus current po translations.